thirdeye/thirdeye-frontend/app/pods/manage/explore/route.js

```diff
@@ -106,16 +116,16 @@ export default Route.extend(AuthenticatedRouteMixin, {
       notifications.error('Retrieving subscription groups failed.', error, toastOptions);
     }
 
-    let subscribedGroups = "";
+    let subscribedGroups = '';
     if (typeof get(this, 'subscriptionGroups') === 'object' && get(this, 'subscriptionGroups').length > 0) {
       const groups = get(this, 'subscriptionGroups');
       for (let key in groups) {
-        if (groups.hasOwnProperty(key)) {
+        if (key in groups) {
           let group = groups[key];
```

Comment:
nitpic: isn't this redundant?
i.e. won't `(key in groups)` always be _**true**_ inside of `for (let key in groups)`. ?

Reply:
I've removed the `key in groups` but `for-each` cannot be used as we are working with object, not array.

Reply:
You could still do `Object.keys(groups).forEach(group) { ...} `. no?

Reply:
I realized I was trying to correct the `eslint` error in an incorrect way. I should be doing `if ({}.propertyIsEnumerable.call(groups, key))`. Using `Object.keys(groups).forEach(group) { ...} ` will work, but it involves usage of extra memory for creating the array and an O(N) processing,